UnityProject/Assets/Scripts/Objects/Nuke.cs

```diff
@@ -148,6 +148,7 @@ private void Detonate()
 			}
 			else
 			{
+				GameManager.Instance.RoundEndTime = 15;
```

Comment:
uhm? What's this?

Reply:
Makes the round time not too long for nuke ending

Reply:
normally what's the ending time?

Reply:
but is not needed technically

Reply:
Yeah, unless you want people to chat a while before starting the next one

Reply:
doesn't the video hide the chat UI?

Reply:
shall remove it

Reply:
ye